code/modules/reagents/reagent_containers/glass.dm

```diff
@@ -279,3 +279,27 @@
 		slot_equipment_priority.Insert(index, slot_head)
 		return
 	return ..()
+
+/obj/item/weapon/reagent_containers/glass/beaker/waterbottle
+	name = "bottle of water"
+	desc = "A bottle of water filled at an old Earth bottling facility,"
+	icon = 'icons/obj/drinks.dmi'
+	icon_state = "smallbottle"
+	item_state = "bottle"
+	list_reagents = list("water" = 48, "fluorine" = 2)//see desc, don't think about it too hard

```

Comment:
Water fluoridation is not toxic and it's not added to water in its elemental form.

Just don't put toxin in the vending machines. Make it have toxic water if you emag it or something